app/src/main/java/org/glucosio/android/GlucosioApplication.java

```diff
@@ -42,7 +42,7 @@
 
 import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
 
-public class GlucosioApplication extends Application {
+public class GlucosioApplication extends MultiDexApplication {
```

Comment:
Why do we need it here? Did we already hit 65k limit?

Reply:
That was a merge mistake. No need in multidex with proguard actually.